src/plugins/interactive_setup/public/cluster_configuration_form.tsx

```diff
@@ -290,40 +300,169 @@ export const ClusterConfigurationForm: FunctionComponent<ClusterConfigurationFor
 };
 
 export interface CertificatePanelProps {
-  certificate: Certificate;
+  certificate: Omit<Certificate, 'raw'>;
+  compressed?: boolean;
+  type?: 'root' | 'intermediate';
+  onClick?(): void;
 }
 
-export const CertificatePanel: FunctionComponent<CertificatePanelProps> = ({ certificate }) => {
+export const CertificatePanel: FunctionComponent<CertificatePanelProps> = ({
+  certificate,
+  onClick,
+  type,
+  compressed = false,
+}) => {
   return (
-    <EuiPanel color="subdued">
+    <EuiPanel color={compressed ? 'subdued' : undefined} hasBorder={!compressed}>
       <EuiFlexGroup responsive={false} alignItems="center" gutterSize="m">
         <EuiFlexItem grow={false}>
           <EuiIcon type="document" size="l" />
         </EuiFlexItem>
         <EuiFlexItem>
-          <EuiTitle size="xxs">
-            <h3>{certificate.subject.O || certificate.subject.CN}</h3>
-          </EuiTitle>
-          <EuiText size="xs">
-            <FormattedMessage
-              id="interactiveSetup.certificatePanel.issuer"
-              defaultMessage="Issued by: {issuer}"
-              values={{
-                issuer: certificate.issuer.O || certificate.issuer.CN,
-              }}
-            />
-          </EuiText>
+          <EuiFlexGroup responsive={false} gutterSize="none" justifyContent="spaceBetween">
+            <EuiFlexItem>
+              <EuiTitle size="xxs">
+                <h3>{certificate.subject.O || certificate.subject.CN}</h3>
+              </EuiTitle>
+            </EuiFlexItem>
+            {!compressed && (
+              <EuiFlexItem grow={false}>
+                <EuiBadge>
+                  {type === 'root'
+                    ? i18n.translate('interactiveSetup.certificatePanel.rootCertificateAuthority', {
```

Comment:
__nit:__ if possible let's use `FormattedMessage` instead of `i18n.translate` (a bit of context: we added `i18n.translate` just to support cases where `FormattedMessage` is not an option - server side messages, Angular etc., it misses some of the functionality that `FormattedMessage` provides as well)